documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_2_Identity_and_Trust_Management.md

```diff
@@ -0,0 +1,113 @@
+# Identity and Trust Management
+
+The International Data Spaces allow participants a cross-company data exchange. In many cases, the participants intending to exchange data have no prior knowledge about the other company and its utilized components to properly assess the consequences of such a data exchange. Thus, the IDS offers mechanisms to gain reliable information which help to establish trust and enable participants to make sovereign and informed decisions.
+Identity and trust management is rooted in the components described in ([Chapter 3.5.1](/documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_1_Onboarding.md)).
+
+## Identities for Devices
+
+The IDS Connector is the central device to establish trust on a technical level and to ensure a secure data exchange across domain boundaries.
+In the IDS, each connector instance possesses it's own identity. Each connector instance is made up of several aspects:
+* The platform the IDS Connector instance depends on. A platform consists of hardware, firmware, operating system and (container) runtime environment.
+* The Connector Core Services software artefacts that provide management functionality and IDS interoperability.
+* The configuration of a IDS Connector (defined data routes, configured Usage Control framework).
+* The Data Apps or other services (e.g., Clearing House services) that are bound to this connector instance.
+
+The IDS Connector identity serves to uniquely identify one instance of a service and app bundle on qualified platforms. The identity concept is equally used for all technical components in the IDS.
+
+The service instance for a connector is typically a Service including core services, some usage control framework and applications (Data Apps).
+Other components (Broker, DAPS, ...) are represented by their Service (represented by one or multiple containers) running on a comparable platform.
+
+One component always is characterized by the combination of platform and service instances. As an example, this Connector instance is running several data apps. The identity is comprised of the platform, the Connector Core Services and the deployed Data Apps.
+
+![Components SW Stack](./media/SW_Stack_Components_connector_blueprint.png)
+#### _Figure 4.1.2.1: Components of the Software Stack of an IDS Connector_
+
+### Component Identifier
+
+The identity of a combination of platform and service instance is bound to an identifier for the service instance.
+
+* Each component gets a unique identified (C_UID) bound to the service instance.
+* The uniqueness of this identifier is ensured by the Identity Provider.
+* Each UID is mapped to a Connector Instance Key (CIK) pair which is typically used for TLS but possibly also data signing and other identity proofs.
```

Comment:
What if the TLS certificates of multiple components of a connector does not match the IDS cert provided by the identity provider? This strongly depends on the architecture and deployment of a connector.